packages/tailwindcss/src/compat/selector-parser.ts

```diff
@@ -369,7 +371,7 @@ export function parse(input: string) {
             ast.push(node)
           }
         }
-        buffer = String.fromCharCode(currentChar)
```

Comment:
I was thinking we did this in purpose for perf reasons but it might not actually make a difference compared to everything else.